samples/packages/enrichment-migrator/src/main/kotlin/EnrichmentMigrator.kt

```diff
@@ -157,6 +161,19 @@ object EnrichmentMigrator {
         return databaseNames
     }
 
+    @JvmStatic
+    fun getConnectionName(connectionQN: String): String {
+        val connection =
+            Connection.select()
+                .where(Asset.QUALIFIED_NAME.eq(connectionQN))
+                .stream()
+                .findFirst()
+                .getOrElse {
+                    throw NotFoundException(ErrorCode.ASSET_NOT_FOUND_BY_QN, connectionQN, "Connection")
+                }
+        return connection.name
```

Comment:
This is the easier way to deal with `Optional`s in Kotlin (imho) — otherwise looks good 👍